_posts/2018-03-23-cache-breaking-for-faster-web-application-updates.md

```diff
@@ -10,26 +10,28 @@ tags: []
 ---
 
 ### When Production Is Broken, It Needs to Be Fixed Now!
-Recently, a 3rd party changed the URL to one of their map services that we use in several of our applications. This came as a bit of a surprise to us so we quickly updated our applications and deployed new versions. Everything seemed great and we continued on our way. However, a full two days later, one of our clients came to me reporting that he still had users that did not have the updated applications in their browsers. The need to teach clients how to clear their browser caches feels unprofessional and it is unnecessary for them and me. So I started doing some digging on how this problem could be solved.
+Recently, the URL changed for one of the map services we use in several of our applications. This came as a bit of a surprise to us, so we quickly updated our applications and deployed new versions. Everything seemed great and we continued on our way. However, two days later one of our clients reported that some of his users still did not have the updated applications in their browsers. Walking clients through how to clear their browser caches felt unprofessional—like I'd be giving them an unnecessary, and possibly unwelcome, explanation of a simple subject. (Obviously not the ideal solution.) So I started doing some digging on how I could solve this problem behind the scenes. 
```

Comment:
Regarding those last couple revised sentences: Is that accurate? Does that work? 

Also consider the following rewording (but, again, is this accurate?): "The obvious solution seemed that clients should clear their browser caches. But walking them through this process felt unprofessional-like I'd be giving them an unnecessary, and possibly unwelcome, explanation of a simple subject. (Obviously not the ideal solution.) So I started doing some digging on how I could solve this problem behind the scenes."